### PR TITLE
URL as labels

### LINKED ISSUE #1: generate images side by side
instead of on top of eachother.  Also, add 10px padding in between

### PR DESCRIPTION
**Pull request by dawnerd**
Fixes #1 and #4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
     "glob": "~3.2.6",
     "optimist": "~0.6.0",
     "progress": "~1.0.0",
-    "mkdirp": "~0.3.5"
+    "mkdirp": "~0.3.5",
+    "load-grunt-tasks": "~0.1.0",
+    "grunt": "~0.4.1",
+    "grunt-contrib-jshint": "~0.6.4",
+    "grunt-contrib-watch": "~0.5.3",
+    "imagick": "git+ssh://git@github.com:tjfontaine/node-imagick.git"
```
Comment (jgallen23) [inline package.json]: this shouldn't be a devDependency

Comment (dawnerd) [inline package.json]: imagick?

Comment (jgallen23) [inline package.json]: devDependencies should only be for grunt and mocha type things.  Just things you need for development.  dependencies are things you'll need to make multishot work.

Comment (dawnerd) [inline package.json]: Gotcha. Just checked in a change for this.

**Comment by dawnerd** [comment]
Now part of #10

**Pull request closed without merging**